assets/styles/_print.scss

```diff
@@ -0,0 +1,84 @@
+@media print {
+  *,
+  *:before,
+  *:after {
+    background: transparent !important;
+    color: #000 !important;
+    /* Black prints faster */
+    box-shadow: none !important;
+    text-shadow: none !important;
+  }
+
+  [is='flynt-navigation-main'],
+  [is='flynt-navigation-burger'],
+  [is='flynt-navigation-footer'],
+  [is="flynt-block-cookie-notice"],
+  .button,
+  footer {
```

Comment:
Instead of hiding individual components, I would suggest to hide the wrapping sections of the layout: `.mainHeader`, `.mainFooter`, `.wpFooter`.
Also, I think buttons should still be visible and not every `footer` element should be hidden, as it might be used differently in a context where you want it to be visible.